### DISCUSSION TITLE
Add support to render ansible-lint output in structured format (JSON)

### DISCUSSION THREAD

**Discussion by ganeshrn** [ansible/ansible-lint#1663] [Ideas] [answered]
Currently, ansible-lint renders output on plain text format making it a bit inconvenient to parse the output and extract the required information. If the output is rendered in JSON format it will be easier for tools invoking ansible-lint (example `ansible-language-server`) to extract information.

**Comment by ssbarnea** [comment, marked as answer]
That is already implemented as `ansible-lint -f codeclimate 2>/dev/null` as codeclimate output is JSON! 

I know that the feature is not easily discoverable but that is because it came from a user asking for codeclimate compatible output, which is a particular JSON format.

Probably you can fix by making a change to document that on CLI output better.

**Reply by ganeshrn** [reply]
Thanks! I missed it. Agreed on the enhancement on CLI front to make JSON support more evident. Should an issue be raised to track the CLI changes?

**Reply by ssbarnea** [reply]
So small that it does not deserve a ticket, but I will not close it if you create one ;)

On the other hand, if you dislike what codeclimate supports, we can easily add another output just for vscode. This could prove useful if we have an already existing generic format that vscode expects. Adding a new formatter is extremely easy.
